src/Api/Program.cs

```diff
@@ -26,7 +25,10 @@ public static IWebHost BuildWebHost(string[] args)
             IConfigurationBuilder builder)
         {
             builder.SetBasePath(Directory.GetCurrentDirectory());
-            builder.AddJsonFile($"appsettings.json", optional: false, reloadOnChange: true);
+#if DEBUG
+            builder.AddJsonFile($"appsettings.json", optional: true, reloadOnChange: true);
```

Comment:
Why don't we need the appsettings.json when not running in DEBUG mode?

Reply:
everything needs to be set in the Env-Variables. The appsettting file is not accessible in the docker image.

Reply:
You could mount it but that's fine for me.